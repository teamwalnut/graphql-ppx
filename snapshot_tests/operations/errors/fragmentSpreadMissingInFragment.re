[%graphql
  {|
  fragment ListFragment on Lists {
    nullableOfNullable
    ...MissingFragment
  }
|}
];
