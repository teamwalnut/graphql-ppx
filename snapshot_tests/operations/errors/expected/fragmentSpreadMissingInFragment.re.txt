
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/fragmentSpreadMissingInFragment.re[0m:[2m5:5-22[0m

  3 [2mâ”‚[0m   fragment ListFragment on Lists {
  4 [2mâ”‚[0m     nullableOfNullable
  [1;31m5[0m [2mâ”‚[0m     [1;31m...MissingFragment[0m
  6 [2mâ”‚[0m   }
  7 [2mâ”‚[0m |}

  [1;33mThe module or file MissingFragment can't be found.[0m
  - If it's a third-party dependency:
    - Did you list it in bsconfig.json?
    - Did you run `rescript build` instead of `rescript build -with-deps`
      (latter builds third-parties)?
  - Did you include the file's directory in bsconfig.json?

